Dockerfile

```diff
@@ -1 +1,68 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
 
+#
+# Basic Parameters
+#
+ARG ARCH="amd64"
+ARG OS="linux"
+ARG VER="2.3.1"
+ARG PKG="grafana-reporter"
+ARG SRC="github.com/IzakMarais/reporter"
+ARG GO_VER="1.16.7"
+ARG GO_SRC="https://golang.org/dl/go${GO_VER}.${OS}-${ARCH}.tar.gz"
+ARG TEX_SRC="https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz"
+ARG UID="root"
+
+#
+# Some important labels
+#
+LABEL ORG="Armedia LLC"
+LABEL MAINTAINER="Armedia Devops Team <devops@armedia.com>"
+LABEL APP="Grafana Reporter"
+LABEL VERSION="${VER}"
+
+#
+# Set the Go environment
+#
+ENV GOROOT="/usr/local/go"
+ENV GOPATH="/go"
+ENV PATH="${PATH}:${GOROOT}/bin"
+
+#
+# Install the requisite packages for the build and runtime
+#
+RUN yum -y update && \
+    yum -y install \
+        git \
+        texlive-collection-basic \
+        texlive-collection-fontsrecommended \
+        texlive-collection-htmlxml \
+        texlive-collection-latex \
+        texlive-collection-latexrecommended \
+        texlive-collection-xetex
+
+#
+# Download and install go
+#
+RUN curl -L "${GO_SRC}" -o - | tar -C "/usr/local" -xzf -
+
+#
+# Build the reporter
+#
+WORKDIR "${GOROOT}"
+
+RUN go get "${SRC}/cmd/grafana-reporter"
+RUN go install -v "${SRC}/cmd/grafana-reporter@v${VER}"
+RUN cp -vi "${GOPATH}/bin/grafana-reporter" "/usr/local/bin"
+
+#
+# Copy the main executable over
+#
+COPY /startup.sh /
+RUN chmod ug+rwx,o-rwx /startup.sh
+
+USER        ${UID}
+EXPOSE      8686
+VOLUME      [ "/templates" ]
+SHELL       [ "/bin/bash", "-c" ]
```

Comment:
This is the first time I am seeing this `SHELL` instruction... What does it do?

Reply:
Declares the default shell to be used when someone opens a... ._shell_ ... to the container without specifying the executable.

Reply:
Possibly... What's your take on this issue @JonHolman ?

Reply:
For the record: I don't feel strongly about this setting either way, which is why the other images don't include it... I just added it here b/c it's transplanted from the developer's example Dockerfile.

Reply:
I agree that we should be consistent.  I don't know if this needs to be set.  Can we go without it until we see a reason to add it (then add it to all, or at a base image level?).

Reply:
Sure. No sweat.